### PR TITLE
SL-998 enable db migration in pipeline

### PR DESCRIPTION
**Pull request by kpakur**
Liquibase migrations have been run during the service start. This has now been configured to be run from the pipeline. During service start the db migration will not be run.

The migration during service start still is run when application is run on the local developer machine, so it is easy for developer to work on code and db changes, this is expected.

To make the change working, azure key vault has been configured and and new elements of the pipeline modules enabled.

This change has been covered in Jira in SL-998.

### CODE REVIEW

File: src/main/resources/liquibase.properties [hunk 1]
```diff
@@ -0,0 +1,5 @@
+changeLogFile=src/main/resources/db/db-changelog.yaml
```
Comment (r-kruk-kainos) [inline src/main/resources/liquibase.properties]: What about test data? Will we add another step for that or do we want to approach it  differently?

File: src/main/resources/liquibase.properties [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+url=jdbc:postgresql://localhost:5453/snl
```
Comment (r-kruk-kainos) [inline src/main/resources/liquibase.properties]: better port please

File: liquibase.gradle [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+  classpath sourceSets.main.runtimeClasspath
+  classpath configurations.liquibase
+  main = "liquibase.integration.commandline.Main"
+
+  def urlString = project.hasProperty("dburl") ? "jdbc:postgresql://$dburl" : liquibaseProps.getProperty('url')
+  def user = project.hasProperty("flyway.user") ? "${rootProject.properties['flyway.user']}" : liquibaseProps.getProperty('username')
+  def password = project.hasProperty("flyway.password") ? "${rootProject.properties['flyway.password']}" : liquibaseProps.getProperty('password')
+
+  args "--changeLogFile=./src/main/resources/db/db-changelog.yaml"
```
Comment (r-kruk-kainos) [inline liquibase.gradle]: If we would like to have some test data on environments after deployment (e.g. preview) how we would like to approach it? Another gradle task for test data or maybe some logic on this task to check if it's not production evnironment?

File: src/main/resources/liquibase.properties (hunk 2 above)
Comment (r-kruk-kainos) [reply src/main/resources/liquibase.properties]: As discussed it could probably be deleted

File: src/main/resources/liquibase.properties (hunk 1 above)
Comment (kpakur) [reply src/main/resources/liquibase.properties]: on preview environments there will be leftovers from functional tests, so any data will be from there if needed.

**Pull request merged**